client/state/data-layer/wpcom/gravatar-upload/index.js

```diff
@@ -73,7 +76,10 @@ export function announceFailure() {
 			{ type: GRAVATAR_UPLOAD_REQUEST_FAILURE }
 		),
 		errorNotice(
-			translate( 'Hmm, your new profile photo was not saved. Please try uploading again.' )
+			translate( 'Hmm, your new profile photo was not saved. Please try uploading again.' ),
```

Comment:
Not part of this PR but I think this could be improved. Why was the profile photo not saved? The network response doesn't seem to contain an error message (it's empty for me) so should probably handled on the API side of things.

Reply:
Definitely 👍  As discussed with @noahtallen in https://github.com/Automattic/wp-calypso/pull/49914#pullrequestreview-588179235 this makes sense to be improved, which is why I opened #50031. Feel free to add any details there 👍